models/resolver.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
fyi @pb82 model for Resolver. See https://github.com/aerogear/data-sync-server/pull/9/files#diff-5709bb01e7a28a79644cde19374931d6R42 for example of what gets inserted